tests/Unit/Domain/Creators/Test_Cylinder.cpp

```diff
@@ -55,79 +55,83 @@ void test_cylinder_construction(
        Direction<3>::upper_zeta()}});
   std::vector<DirectionMap<3, BlockNeighbor<3>>> expected_block_neighbors{};
   std::vector<std::unordered_set<Direction<3>>> expected_external_boundaries{};
+  using TargetFrame = Frame::Inertial;
```

Comment:
Do you have a way of creating a graphic that shows the block order/orientations? If you have an easy way to do it that would be a great addition to the documentation, otherwise I can do it at a later time.

Reply:
I will try to come up with something. Thank you for your comments. 
About my earlier comment though, do the corners need a particular order to be in? I have printed the coordinates of the first and second face derived in `physical_separation()`and noticed that for periodic boundary conditions, I get `x - x', y-y'` from Block A to B, but  `x-y', y-x'`from Block B to Block A.

Reply:
Hi Tom, I sent you a slack message regarding `physical_separation` - however I see something I didn't address. The corners should be listed this order for each block:

```
0: lower xi, lower eta, lower zeta
1: upper xi, lower eta, lower zeta
2: lower xi, upper eta, lower zeta
3: upper xi, upper eta, lower zeta
4: lower xi, lower eta, upper zeta
5: upper xi, lower eta, upper zeta
6: lower xi, upper eta, upper zeta
7: upper xi, upper eta, upper zeta
```

For identified faces, it should work as in the tesseract example. Domains are pretty visual, so I don't mind jumping on a call sometime if you want to show me what the set up looks like. Have you tried running a scalar wave with this domain or using the ExportCoordinates executable? It can help you figure out if everything is set up correctly.

Reply:
Thanks! Yes, I have used it in both ExportCoordinates as well as in `SolveHalfSpaceMirror` from my other [PR.](https://github.com/sxs-collaboration/spectre/pull/2298) I will contact you on Slack so that we might agree on a time for a call.